core/src/main/java/hudson/Proc.java

```diff
@@ -505,6 +505,7 @@ public OutputStream getStdin() {
     /**
      * Debug switch to have the thread display the process it's waiting for.
      */
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
     public static boolean SHOW_PID = false;
```

Comment:
Probably could use a system property, unsure whether it's something you want on startup.